### ISSUE TITLE
Default deserializer performance

### ISSUE THREAD

**Issue by jsonhero** [subframe7536/kysely-sqlite-tools#7]
The line here is a very performance heavy operation:
https://github.com/subframe7536/kysely-sqlite-tools/blob/master/packages/plugin-serialize/src/serializer.ts#L31

This seems far too aggressive for default behavior. Could I suggest that date parsing is opt-in? Let alone recursive date parsing that checks for a date on every string.

Also, wouldn't doing something like this to check for objects be more intuitive? JSON.parse is an expensive operation, and shouldn't be getting executed for every arbitrary string value.

```
if (parameter.startsWith('{') && parameter.endsWith('}')) {
          return JSON.parse(parameter)
}
```

**Comment by subframe7536** [comment]
Yeah, default serializer/deserializer is "just ok to run" and have much room to optimize for performance. Thanks for you suggestion and I will try to improve it.

you can write your own serializer/deserializer currently

**Comment by subframe7536** [comment]
maybe https://github.com/unjs/destr is a better choice

**Comment by subframe7536** [comment]
complete in V5.7
